packages/math-input/src/components/two-page-keypad.js

```diff
@@ -2,23 +2,24 @@
  * A keypad with two pages of keys.
  */
 
-const React = require("react");
+const {StyleSheet} = require("aphrodite");
 const PropTypes = require("prop-types");
+const React = require("react");
 const {connect} = require("react-redux");
```

Comment:
Since you changed other `require()`s to `import`s, could you do the same here?

Reply:
I started to do that but then I started getting errors because some of these `require()` statements import files that still use `module.exports = ...`.  I think I'd like to do a whole sweep of moving away from CJS exports to ES6 imports.  But the scope feels a bigger than this PR.